pyodide-build/pyodide_build/testing.py

```diff
@@ -1,10 +1,13 @@
 import contextlib
 import inspect
 from base64 import b64encode
+from sys import version_info
 from typing import Callable, Collection, Optional
 
 import pytest
 
+PYVERSION = f"python{version_info.major}.{version_info.minor}"
```

Comment:
I think tests should use Pyodide Python version not local Python version, though currently our assumption is that those two are strictly same.

Reply:
Yeah. This is easier to write though. What should I do to get the Pyodide version here?

Reply:
Well I think tests that require knowing python version can just do `version = pyodide.runPython('from sys import version_info, ...')` each time like you did at the `pyodide.ts`.
If there are too many of those, maybe we could expose a version variable to js such as `pyodide.pythonVersion` (I'm not sure though)

Reply:
We could just add it as a attribute to selenium e.g., `selenium.pyversion`

Reply:
Okay, how about this?

Reply:
Yeah, this looks better. In the long term, I think we should split out testing-related things from pyodide-build, but that could be discussed in other place.